src/Http/Controller/AdvsController.php

```diff
@@ -869,6 +869,22 @@ public function getLocations()
         return $location;
     }
 
+    public function  multipleOperations(Request $request){
+        $ads_array = $request->input('check');
+        $action = $request->input('action');
+        if(!is_null($ads_array)){
+            if ($action == 'delete'){
+                $this->adv_model->newQuery()->whereIn('advs_advs.id', $ads_array)->delete();
+                return redirect()->route('profile::ads')->with('alert_message',  trans('visiosoft.module.advs::field.deleted'))->with('status', 'success');
+            } elseif ($action == 'extend') {
+                $newDate = date('Y-m-d H:i:s', strtotime(date('Y-m-d H:i:s') . ' + ' . setting_value('visiosoft.module.advs::default_published_time') . ' day'));
+                $this->adv_model->newQuery()->whereIn('advs_advs.id', $ads_array)->update(['finish_at' => $newDate]);
+                return redirect()->route('profile::ads')->with('alert_message', trans('visiosoft.module.advs::field.extended'))->with('status', 'success');
+            }
+        }
+        return redirect()->route('profile::ads')->with('alert_message',  trans('visiosoft.module.advs::message.error_select_ad'))->with('status', 'error');
```

Comment:
just return one time and use a message variable.

`return redirect()->route('profile::ads')->with('alert_message', $msg)->with('status', 'success');`